pkg/agent/route/interfaces.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package route
+
+import (
+	"net"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+)
+
+// RouteService is the interface for routing container packets in host network.
+type RouteService interface {
+	Initialize(nodeConfig *config.NodeConfig) error
+
+	Reconcile(podCIDRs []string) error
+
+	// AddRoutes adds routes to the provided podCIDR.
```

Comment:
please include in the comment the behavior if the routes already exist

Reply:
Done